### DISCUSSION TITLE
[Help]toc button don't works in 0.64.x releases

### DISCUSSION THREAD

**Discussion by sudoexec** [outline/outline#3585] [Self Hosting] [answered]
In 0.63.0, the toc button works well.
After upgrading to 0.64.0/0.64.1/0.64.2, the toc button never works and not toc toggles.

**Comment by tommoor** [comment, marked as answer]
More issues than usual in this release 😢 – the issue is fixed in `v0.64.3` which is building right now.

**Reply by sudoexec** [reply]
Thanks for your great work.
Now it works in v0.64.3.
